### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by tfiedlerdejanze**
- [x] Please [sign Adobe's CLA](http://opensource.adobe.com/cla.html) if this is your first time contributing to an Adobe open source repo. Thanks!

### CODE REVIEW

File: lib/style/module_directives.ex [hunk 1]
```diff
@@ -333,8 +333,8 @@ defmodule Styler.Style.ModuleDirectives do
 
     if Enum.any?(liftable) do
       # This is a silly hack that helps comments stay put.
-      # the `cap_line` algo was designed to handle high-line stuff moving up into low line territory, so we set our
-      # new node to have an abritrarily high line annnnd comments behave! i think.
+      # The `cap_line` algorithm was designed to handle high-line stuff moving up into low line territory,
+      # so we set our new node to have an arbitrarily high line so comments stay where they are.
```
Comment (novaugust) [inline lib/style/module_directives.ex:337]: i'm happy to take typo fixes, but prefer my comments to myself remaining my comments to myself in tone and style, including nonsensical things like "annnnnd"

File: test/style/pipes_test.exs [hunk 2]
```diff
@@ -482,7 +482,7 @@ defmodule Styler.Style.PipesTest do
   end
 
   describe "simple rewrites" do
-    test "rewrites anon fun def ahd invoke to use then" do
+    test "rewrites anonymous function invocation to use then" do
```
Comment (novaugust) [inline test/style/pipes_test.exs:485]:
```suggestion
    test "rewrites anonymous function invocations to use then" do
```

**Review by novaugust — CHANGES_REQUESTED** [review]
cheers for the PRs as always

i'm happy to take typo fixes, but would prefer my comments to myself remaining my comments to myself in tone and style, including nonsensical things like "annnnnd" and other silly notes
if these were public docs, unquestionably yours is better and must be taken. but for the internals of the library, i'm happy to have it read like my cozy little nook that makes me laugh when i come back to it :) hope you understand, appreciate the efforts regardless

**Comment by tfiedlerdejanze** [comment]
> prefer my comments to myself remaining my comments to myself in tone and style

Absolutely fair and not very considerate on my end to change a comment like that in the first place. Plus, i enjoy reading these comments as they are! :)

**Comment by novaugust** [comment]
dawwwh you're too sweet. thanks theodor!

**Pull request merged**